assets/out

```diff
@@ -100,6 +103,7 @@ set_overriden_values() {
     value=${kv#*=}
     helm_cmd="$helm_cmd --set '$kv'"
     helm_echo="$helm_echo --set '$key=***HIDDEN***'"
+    helm_diff_cmd="$helm_cmd --set '$kv'"
```

Comment:
shouldn't the be `$helm_diff_cmd` on the right?

Reply:
Good catch—fix incoming!